browser/ui/webui/ai_chat/ai_chat_ui.cc

```diff
@@ -79,13 +102,25 @@ void AIChatUI::BindInterface(
     embedder_->ShowUI();
   }
 
+  content::WebContents* web_contents = nullptr;
+  bool is_standalone = false;
 #if !BUILDFLAG(IS_ANDROID)
   browser_ = ai_chat::GetBrowserForWebContents(web_ui()->GetWebContents());
   DCHECK(browser_);
+  TabStripModel* tab_strip_model = browser_->tab_strip_model();
+  DCHECK(tab_strip_model);
+  web_contents = tab_strip_model->GetActiveWebContents();
+  is_standalone = tab_strip_model->GetIndexOfWebContents(
+                      web_ui()->GetWebContents()) != TabStripModel::kNoTab;
+#else
+  web_contents = GetActiveWebContents(profile_);
+  is_standalone = web_contents == nullptr;
```

Comment:
Shouldn't the `is_standalone` for both platforms be based on whether this WebUI _is_ the active Tab vs a separate WebContents, i.e. for both platforms:
```cpp
is_standalone = web_contends !== web_ui->GetWebContents()
```
?

Reply:
yeah perhaps it's better to do that way for the feature standalone option on Android

Reply:
actually that's not right as `web_contents` is the current page contents vs `web_ui->GetWebContents()` is an actual web ui with the chat contents. So I think Android detection is correct it means we have a chat opened only if we don't have an active tab and the web ui, but for desktop I'm not sure exactly what do we check there, we are looking for an index of the char web ui and we assume it's not a standalone if we found it. As to me the check should be
`is_standalone = web_contents == web_ui->GetWebContents();`